pkg/tfgen/generate_nodejs.go

```diff
@@ -652,7 +652,7 @@ func (g *nodeJSGenerator) emitNPMPackageMetadata(pack *pkg) error {
 			"build": "tsc",
 		},
 		DevDependencies: map[string]string{
-			"typescript": "^2.6.2",
+			"typescript": "^2.9.2",
```

Comment:
I can revert this before checking in, but I wanted some change that would cause a diff so we could see what happened.